web/styles/zulip.css

```diff
@@ -1138,9 +1138,9 @@ td.pointer {
     word-wrap: break-word;
     cursor: pointer;
     border: none;
-    /* The left padding value maintains the same amount
-      of space to the left of the message box from before
-      the unread marker was a grid item. */
+    /* The left padding value accounts for a 2px
+      unread marker, ensuring a uniform 5px of
+      padding on either side of the message box. */
     padding: 0 5px 0 3px;
```

Comment:
I wonder if as a follow-up we want to make this a calculation involving a constant for the width of the unread marker, in that there's a good chance we might change that value to 1px.

Reply:
That's a possibility. I think the other issue is that we might want the padding value to change based on whether the unread marker is even present, so that especially when the active-message marker is in place, the padding on both sides of the message box is identical (the blue box looks a lot tighter on the marker side, at least to my eyes).

Shall I keep this on a list for a future PR?

Reply:
Yeah let's leave this for a follow-up.